miden-tx/src/tests/kernel_tests/test_asset_vault.rs

```diff
@@ -331,15 +361,18 @@ fn test_remove_fungible_asset_success_balance_remaining() {
     );
 
     assert_eq!(
-        process.get_mem_value(ContextId::root(), memory::ACCT_VAULT_ROOT_PTR).unwrap(),
```

Comment:
using the `read_root_mem_value` utility to read memory on all tests. it makes the test case shorter and easier to read.